lib/cdo/aws/s3_packaging.rb

```diff
@@ -39,7 +39,7 @@ def update_from_s3
     begin
       ensure_updated_package
     rescue Aws::S3::Errors::NoSuchKey
-      @logger.info 'Package does not exist on S3. If you have made local changes to code-studio, you need to set build_code_studio and use_my_code_studio to true in locals.yml'
+      @logger.info "Package does not exist on S3. If you have made local changes to #{@package_name}, you need to set build_#{@package_name} and use_my_#{@package_name} to true in locals.yml"
```

Comment:
It's a bit unfortunate that `blockly-core` doesn't require a `use_my`, just a `build` ... it makes this line potentially misleading.

I'm not sure how worried I am about that, because setting a useless `use_my_blockly_core` isn't harmful.